Regex_formData.rb

```diff
@@ -0,0 +1,13 @@
+# regex to get name , email & phone-number from a string
+
+formData = "My name is Rohan Dhalpe gender: M. You can contact me at email: rohan.dhalpe@joshsoftware.com or my work phone: 8010317758 i get paid $5000"
```

Comment:
Never add original data, esp numbers.
Also check variable naming conventions in ruby.
formData can be `form_data`.

Reply:
Updated.